js/utime.ts

```diff
@@ -32,7 +17,11 @@ export function utimeSync(
   atime: number | Date,
   mtime: number | Date
 ): void {
-  sendSync(...req(filename, atime, mtime));
+  sendSync(OP_UTIME, {
+    filename,
+    atime: toSecondsFromEpoch(atime),
```

Comment:
I realize this might not be a regression, but to avoid a future addition of a nanosecond-precision API, maybe split this in a [seconds, nanoseconds] tuple.

Reply:
I also think it’s odd that this op takes seconds instead of something more precise but in this PR I’m only trying to change the serialization, not modify the logical messages or behavior of the ops. So I would prefer to do this later.

Reply:
I've added a TODO for this.